README.md

```diff
@@ -12,6 +12,8 @@
 
 ## Statistical Tools (Widgets)
 
+- [GitClear Commit Activity Bubble Graph](https://www.gitclear.com/github_profile_dynamic_readme_free#commit_activity) - Dynamically generating daily visualization of which tickets/repos/branches are being worked on by individual or team
```

Comment:
Instead of [GitClear Commit Activity Bubble Graph] rename it to [GitClear Commit Activity].
If needed you can modify the description & add those wording.